### ISSUE TITLE
Logical properties support

### ISSUE THREAD

**Issue by certainlyakey** [kristerkari/stylelint-declaration-block-no-ignored-properties#79]
Hi and thanks for a great plugin!
It would be good to add support for property combinations that include logical properties. 
Eg right now this combination is reported as a problem: 

```
display: inline;
width: 74px;
```

However, this is not reported: 

```
display: inline;
inline-size: 74px;
```

I use stylelint-declaration-block-no-ignored-properties v2.5 with stylelint 14.5.3 and postcss 8.4.8.

**Comment by kristerkari** [comment]
@certainlyakey Thanks for reporting this! Would you be able to contribute a fix for it?

**Comment by kristerkari** [comment]
Fixed in #82, releasing a new version soon.
